java/ql/src/Security/CWE/CWE-209/SensitiveDataExposureThroughErrorMessage.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @name Information exposure through a error message
+ * @description Information from a error message propagates to an external user.
+ *              Error messages can unintentionally reveal implementation details
+ *              that are useful to an attacker for developing a subsequent exploit.
+ * @kind problem
+ * @problem.severity error
+ * @security-severity 5.4
+ * @precision high
+ * @id java/error-message-exposure
+ * @tags security
+ *       external/cwe/cwe-209
```

Comment:
Perhaps this query should also be tagged with `cwe-497`?

Reply:
Sounds like https://cwe.mitre.org/data/definitions/497.html is broader than https://cwe.mitre.org/data/definitions/209.html and will in most cases also be present: I cant think of a case where https://cwe.mitre.org/data/definitions/209.html is relevant where https://cwe.mitre.org/data/definitions/497.html is not. 

In the end I do not know who can make this decision, but I would not mind adding it.